pkg/git/ssh_config.go

```diff
@@ -22,7 +22,7 @@ import (
 	"path/filepath"
 	"text/template"
 
-	"github.com/pipe-cd/pipecd/pkg/config"
+	configv1 "github.com/pipe-cd/pipecd/pkg/configv1"
```

Comment:
```suggestion
	configv "github.com/pipe-cd/pipecd/pkg/configv1"
```
How about making this like we did in pipedv1 entry
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/pipedv1/cmd/piped/piped.go#L72

Reply:
@khanhtc1202 
This package is for both pipedv0 and pipedv1, so it would be nice to distinguish config and configv1 to avoid misinterpreting them.
So I think it is good as is.
WDYT?

Reply:
Got your point 🙆‍♂️